content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -58,6 +58,61 @@ corresponding reclamation or reconciliation process, as a given admission
 controller does not know for sure that a given request will pass all of the
 other admission controllers.
 
+The ordering of these calls can be seen below.
+
+
```

Comment:
Per other Mermaid diagrams, the svg file is saved in `/docs/images/`.

You can reference here it using the Hugo `{{< figure >}}` [shortcode](https://gohugo.io/content-management/shortcodes/#figure). You also need to add the GitHub gist to the mermaid preview. See a similar [example](https://github.com/kubernetes/website/blob/38a9034e7192712ce3c08334cee33d1a934fa31d/content/en/docs/contribute/style/diagram-guide.md?plain=1#L553).

Reply:
You can use Mermaid as-is, but I would use SVG for this one. It will make things quite a bit easier.

Reply:
In terms of the SVG XML source of the image, I'd put it inside `content/en/docs/reference/access-authn-authz/`

Reply:
That’s a nice suggestion.